build.gradle

```diff
@@ -2,25 +2,25 @@
 buildscript {
     repositories {
         google()
-        jcenter()
         maven {
             url 'https://maven.google.com/'
             name 'Google'
         }
+        jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.1.3'
+        classpath 'com.android.tools.build:gradle:3.2.0-beta01'
     }
 }
 
 allprojects {
     repositories {
-        jcenter()
-        mavenCentral()
         maven {
             url 'https://maven.google.com/'
             name 'Google'
         }
+        mavenCentral()
```

Comment:
Please revert - jcenter dependency should be first.

Reply:
jcenter before the maven was making Android Studio error with common not being resolved, so i switched them and it worked